taskcat/deployer.py

```diff
@@ -472,7 +493,7 @@ def _copy_file(self, in_file, out_file):
         if in_file is not out_file:
             shutil.copyfile(in_file, out_file)
 
-    def aws_api_init(self, aws_profile=None, aws_access_key_id=None, aws_secret_access_key=None):
+    def aws_api_init(self, aws_profile=None, aws_access_key_id=None, aws_secret_access_key=None, aws_session_token=None):
```

Comment:
@avattathil @agamzn 
This function needs to be centralized... maybe in client factory. This is the most advanced one in taskcat though as it supports GovCloud (and now session tokens).